### DISCUSSION TITLE
Company onboarding restriction for VAT Category

### DISCUSSION THREAD

**Discussion by BOTSolutions8686** [lavaloon-eg/ksa_compliance#135] [Q&A] [answered]
Hi . I have onboarded a company with standard VAT catagory
now if I try to submit an invoice for a forigen customer who does not have a vat number the system does not let me . "The error come as "The comapny _____ is registered with Standard Rated VAT which requires VAT number for the client" 

what should I do for this use case. 
As my customer generates ZERO RATED , Forigen invoices and standard invoices. 

Do I need to onboard the same company with different VAT catogories ? 
How will that work ?

**Comment by omarashrafsarhan** [comment]
Hello,

- I need to know the type of transaction you selected in the onboarding, was it both standard and simplified or standard only
- If the error in the TAX category only you can configure new Sales tax template with the Zero rate category but make sure that you selected the correct ZATCA category.
- If you onboarded the company with type of transaction Standard tax invoice only you will have to onboard again.

**Reply by BOTSolutions8686** [reply]
Hi . 
1 - I selected standard only
2- Yes it is no problem to submit an invoice with zero rated VAT and correct category plus EURO as document currency even now. The problem is that some forigen customer do not have VAT number , so when I try to submit the invoice for them, The system prompts "The company is registered for standard Rated VAT and requires a VAT Number or other vat ID" but we do not have any other ID at the moment.  

3- I have onboarded with standard TAX invoice . As I understand billing to forigen company who does not have a number will also be a b2b Standard invoice. Please let me know if I did wrong.

**Reply by omarashrafsarhan** [reply]
The problem here is in the onboarding you created a certificate for the standard tax invoice B2B only, so you will not be able to produce a simplified invoice B2C "This the type you need in this case" as per the document the foreign customer or any transaction without VAT registration number for the customer is simplified tax invoice.

You will need to re-onboard again with selecting type of business transaction "Both" to be able to produce both standard tax invoice and simplified tax invoice but carful this will cause you to reset the ICV and PIH

Or if it okay you can onboard another company with the required type of transaction.

I hope this help.

**Reply by BOTSolutions8686** [reply]
I understand Thank you for detailed explanation. 
Just so I understand it properly - For any invoice that is in forigen currency and the customer does not have VAT number - I will have to generate a b2c invoice or specifically onboard the company with both b2b and b2c

Additional question 
If I have two companies added in erpnext - Same VAT Number but different cr numbers . 
Do I need to onboard both companies separately on will the module work with just one onboarding as the vat number is the same. 

And Thank you for your prompt response it is a great help .

**Reply by omarashrafsarhan** [reply, marked as answer]
Hi,

1. Invoices in ZATCA has two types Simplified B2C (The customer VAT number is not existing), and Standard B2B (The customer VAT number is mandatory or any other IDs e.g. CRN, TIN, ...etc.)
2. Our App support Mutiple company with the same VAT and different CRN each one of them need to be onboarded separately to get its own associated certificate and both of them will work properly on the same server
